cmd/users/main.go

```diff
@@ -284,7 +285,9 @@ func newService(db *sqlx.DB, tracer opentracing.Tracer, auth mainflux.AuthServic
 		logger.Error(fmt.Sprintf("Failed to configure e-mailing util: %s", err.Error()))
 	}
 
-	svc := users.New(userRepo, groupRepo, hasher, auth, emailer)
+	uuidProvider := uuid.New()
```

Comment:
Since we are using `hasher` as an instance of the hasher (`bcrypt`), we should do the same for the ID provider. We don't really care what's the underlying implementation.